src/app/pages/MainArticlesScreen/components/ArticleComponent/index.ts

```diff
@@ -1,7 +1,10 @@
 import React from 'react';
 import { compose } from 'redux';
 import { injectIntl } from 'react-intl';
+import { NavigationScreenComponent, withNavigation } from 'react-navigation';
 
 import ArticleComponent from './Article.component';
 
-export const Article = compose(injectIntl)(ArticleComponent) as React.ReactType;
+export const Article = withNavigation(
```

Comment:
something bad happaned here. Compose should wrap all HOCS, witH Natvagation is a HOC and should be inside, 
Also ar eyou able to make correct typings to NavigationScreenComponent? Its its something not trivial to fix any can stay :)

Reply:
I tried once before and failed, but I'll try again :)